src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3467,7 +3467,7 @@ def _wait_for_webapp(tunnel_server):
         if tries == 0:
             logger.warning('Connection is not ready yet, please wait')
         if tries == 60:
-            raise CLIError("Timeout Error, Unable to establish a connection")
+            raise CLIError("SSH timeout, your app must be running before it can accept SSH connections. Use `az webapp log tail` to review the app startup logs.")
```

Comment:
I don't think it's a timeout error. Maybe "Exceed maximum retry times"

Reply:
This is just polling for the app status (at 1 second intervals), before making an ssh tunnel connection. This message would be useful for users to understand what went wrong